packages/calcite-components/src/components/tip-manager/tip-manager.scss

```diff
@@ -72,15 +81,14 @@
     p-4;
   animation-name: none;
   animation-duration: var(--calcite-animation-timing);
-  block-size: var(--calcite-tip-manager-height);
+  block-size: var(--calcite-internal-tip-manager-block-size, 19vh);
   &:focus {
     @apply focus-outset;
   }
 }
 
 ::slotted(calcite-tip) {
   @apply m-0 border-none;
-  max-inline-size: var(--calcite-tip-max-width);
```

Comment:
I think this should be `--calcite-tip-manager-max-size-x`

Reply:
Can we talk about this naming to make sure we're all on the same page with renaming width/height as size-x/-y?
@macandcheese @jcfranco

Reply:
I’m not sure this is needed at all. A user can set a desired width directly on the Tip that is slotted. 

That’s how Carousel will function as well which is the replacement component / migration path for these Tip components.

Reply:
This one could just be deprecated I think.

Reply:
I'd agree with deprecation.